lighthouse-core/scripts/run-git3po.sh

```diff
@@ -9,7 +9,7 @@ set -ex
 DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "$DIRNAME"
 
-START_AT=`date +%s -d "1 day ago"`
+START_AT="$(date +%s -d '1 day ago')000"
```

Comment:
```suggestion
START_AT=`date +%s -d '1 day ago'`000
```

easier to parse

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
We know output of date is just numbers

So I feel safe rec backticks here but I will change to approve